axelar/README.md

```diff
@@ -0,0 +1,43 @@
+# Akash
+
+| | |
+|---|---|
+|Version|`v0.34.0`|
+|Binary|`axelard`|
+|Directory|`.axelar`|
+|ENV namespace|`AXELAR`|
+|Repository|`https://github.com/axelarnetwork/axelar-core`|
+|Image|`ghcr.io/akash-network/cosmos-omnibus:v0.4.3-axelar-v0.34.0`|
+
+## Examples
+
+- Run on Akash with the [example deploy.yml](./deploy.yml)
+- Run locally using the [example docker-compose.yml](./docker-compose.yml)
+
+## Chain information
+
+The [Cosmos Chain Registry](https://github.com/cosmos/chain-registry) publishes up to date chain info for Axelar.
+
+|Variable|Value|
+|---|---|
+|`CHAIN_JSON`|`https://raw.githubusercontent.com/axelarnetwork/axelarate-community/main/resources/mainnet/genesis.json`|
```

Comment:
```suggestion
|`CHAIN_JSON`|`https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/chain.json`|
```

Reply:
added in https://github.com/akash-network/cosmos-omnibus/commit/8ec6aec59a2ed21fdf1ccb18670d24ed23a9dbed